### DISCUSSION TITLE
they-say/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#3] [Announcements]
# they-say/

A Picture Is Worth A Thousand Words

https://www.supergoodcode.com/they-say/

**Comment by NHOrus** [comment]
You are kind of an insane people we depend upon.
Please don't strain your brain.

**Comment by atlasvirtualreality** [comment]
Won't it get overheads, if you use this extention for the zink translation layer since we already have native opengl implementation.

**Reply by atlasvirtualreality** [reply]
Also if zink succeed will RadeonSI get deprecated? because as i see things there is still no subsequential performance gains.

**Reply by zmike** [reply]
No and no. RadeonSI will never be deprecated so long as it's still actively developed.
